README.md

```diff
@@ -105,7 +105,7 @@ A `Makefile` is available in the root of the repository to abstract away commonl
 
 **`make format`**
 
-> This will peform a run of `isort` and `black`, this **will** modify files if issues were found
+> This will perform a run of `isort` and `black`, this **will** modify files if issues were found
```

Comment:
```suggestion
> This will perform a run of `ruff`, this **will** modify files if issues were found
```

Reply:
(Same applies to the lint one above this too)

Reply:
Nice catch, thanks!!